crates/zksync_error/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+
+[package]
+name = "zksync_error"
+version = "0.1.0"
+edition = "2021"
+[lib]
+
+[dependencies]
+lazy_static = "1.5.0"
+serde = { version = "1.0.210", features = [ "derive", "rc" ] }
+serde_json = "1.0.128"
+strum = "0.26.3"
+strum_macros = "0.26.4"
+zksync-error-description = { git = "https://github.com/matter-labs/zksync-error", branch = "main"}
+anyhow = "1.0"
```

Comment:
Hmm, is this file auto-generated too? If so at the very least we need a header making this clear but I also think I would prefer if it was left for the user to write